issue-triaging.md

```diff
@@ -44,6 +44,52 @@ between the involved parties to allow work to begin or for the issue to be close
 - Guiding any interested parties to another person or resource that may be more knowledgeable about an issue.
 - Suggesting an issue for discussion at a SIG meeting if a synchronous discussion would be more productive.
 
+It is recommended that a triager should follow these steps when a new issue is reported:
+
+1. First and foremost step is to access the validity of the issue. Here's an rough outline to access the validity:
+   - Does the issue include a clear description of the problem?
+   - Does it provide the user’s configuration, the expected behavior, and any other relevant context?
+   - If it's reported as a bug, are the steps to reproduce clearly outlined?
+
+2. If the issue is deemed valid and requires attention, apply appropriate labels based on its nature:
+   - For bugs, add the `bug` label along with `waiting-for-codeowners`.
+   - For enhancements, use the `enhancement` label and also add `waiting-for-codeowners`.
+   - If the issue is about a new component, add the `sponsor-needed` label. We should encourage user to join our SIG and propose the new component. They can also reach out on #otel-collector on CNCF slack.
+   - Check out [state diagram](#state-diagram) for detailed information about all labels.
+
+3. An issue may be considered invalid for the following reasons:
+   - The user is using an incorrect configuration.
+   - The bug/enhancement has already been addressed in the newer version.
+   - The issue was filed in the wrong repository:
+       - For example, we might receive an issue for `otlpexporter` in contrib. You should transfer the issue to the `opentelemetry-collector` repository.
```

Comment:
We should probably add guidance on how to transfer the issue. I prefer asking project maintainers on CNCF Slack rather than the issue, to avoid noise and confusion for the person who filed the issue, but happy to hear if someone has a different preference.